lib/internal/errors.js

```diff
@@ -234,3 +248,22 @@ function oneOf(expected, thing) {
     return `of ${thing} ${String(expected)}`;
   }
 }
+
+function valueOutOfRange(name, compareWord, target, actual) {
```

Comment:
I think `ERR_INVALID_OPT_VALUE` may not suitable for error here because "size" is not a value in an options object.
If use `ERR_INVALID_OPT_VALUE`, the error message will change to:
```
RangeError [ERR_INVALID_OPT_VALUE] The value "Infinity" is invalid for option "size"
```
Is it OK?